src/lib/components/content/RoundInfo.svelte

```diff
@@ -37,7 +38,7 @@
 
   {#if note}
     <p class="note">
```

Comment:
Small note (pun intended): can we add a whitespace nowrap to the `.note` class so that newlines are respected? I've seen some builds where bullet points are desired, and while we aren't allowing markdown, let's go ahead and at least let people format bullet points and other such things manually

Reply:
We could do something as simple as replacing `\n` with `<br>`, that might do the trick